spec/litmus/basic_dsc_resources/file/dir_valid_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'spec_helper_acceptance_litmus'
+
+def verify_directory_resource(dsc_type, dsc_module, dsc_props)
+  assert_dsc_resource(
+      dsc_type,
+      dsc_module,
+      :Ensure => dsc_props[:dsc_ensure],
+      :Type => dsc_props[:dsc_type],
+      :DestinationPath => dsc_props[:dsc_destinationpath]
+  )
+end
+
+describe 'Apply DSC "File" (directory) resource' do
+
+  dsc_type = 'file'
+  dsc_module = 'PSDesiredStateConfiguration'
+
+  context 'A Directory Resource with Valid "DestinationPath" Specified' do
+    before(:all) do
+      @dsc_destinationpath = 'C:\test_dir'
+      run_shell("remove-item -path C:/temp/test_dir -Force -Recurse -ErrorAction SilentlyContinue", :expect_failures => true)
```

Comment:
Litmus' "run_shell" method doesn't (yet?) have an option for "accept_all_exit_codes", so I've used "expect_failures" here, which is not semantically correct, I don't really expect failure, but I also don't care if there is one.

One solution is to add "accept_all_exit_codes" to Litmus' "run_shell" the same way I added "noop" to "apply_manifest" like in this commit:
https://github.com/puppetlabs/puppet_litmus/commit/5bcbfecb11761be094ce9be2679609db5754d9a2